src/Services/Contracts/ConsultationServiceContract.php

```diff
@@ -26,4 +26,5 @@ public function proposedTerms(int $orderItemId): ?Collection;
     public function setRelations(Consultation $consultation, array $relations = []): void;
     public function setFiles(Consultation $consultation, array $files = []): void;
     public function getConsultationTermsByConsultationId(int $consultationId, array $search = []): Collection;
+    public function getConsultationsListForUser(array $search = []): Builder;
```

Comment:
When I see method named `xxxForUser()` I kind of except $user as a param of this method; otherwise you could probably rename this to getConsultationsForCurrentUser or getConsultationsForAuthenticatedUser or something?